src/index.tsx

```diff
@@ -203,9 +202,7 @@ const content = (
   <MuiThemeProvider theme={theme}>
     <Provider store={theStore}>
       <Router>
-        <AppContainer>
-          <Routes />
-        </AppContainer>
+        <AppContainer />
```

Comment:
The <Routes /> is now in the AppContainer file, where the width and height are set explicitly.